{"name": "Негативный тест авторизации существующего пользователя", "status": "passed", "description": "Негативный тест. Получение  в тело ответа. code == 403 и наличия в ответе кода \"message\": \"User already exists\"", "start": 1725598659223, "stop": 1725598660242, "uuid": "79c3bbc6-611a-492d-a732-8ee21c43a6e5", "historyId": "4654aa6cdd3b277f54df0d45fd4cb639", "testCaseId": "4654aa6cdd3b277f54df0d45fd4cb639", "fullName": "tests.test_registration.TestRegistration#test_repeat_registration_of_the_existing_user", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_registration"}, {"name": "subSuite", "value": "TestRegistration"}, {"name": "host", "value": "LAPTOP-KL5MP4V8"}, {"name": "thread", "value": "3856-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_registration"}]}